apps/fabric8-mq/src/main/java/io/fabric8/mq/coordination/KubernetesControl.java

```diff
@@ -107,27 +112,18 @@ private void populateBrokerStatistics(Pod pod, J4pClient client) {
                 Object brokerId = BrokerJmxUtils.getAttribute(client, root, attribute);
 
                 attribute = "OpenWireURL";
-                Object uri = BrokerJmxUtils.getAttribute(client, root, attribute);
+                Object uriObj = BrokerJmxUtils.getAttribute(client, root, attribute);
+                URI uri = new URI(uriObj.toString());
+                int port = uri.getPort();
+
+                String amqBrokerURI = "tcp://" + KubernetesHelper.getHost(pod) + ":" + port;
```

Comment:
@rajdavies This should use the pod IP, not the host IP. This should be available via `pod.getStatus().getPodIP()`.